create-rust-app_cli/src/qsync/hook.rs

```diff
@@ -214,15 +218,17 @@ impl Hook {
             }
         }
 
-        if query_key.len() > 0 {
+        if !query_key.is_empty() {
             query_key.insert_str(0, ", ")
         }
         query_key.insert_str(0, &format!("{:?}", self.query_key_base));
 
         query_key
     }
+}
 
-    pub fn to_string(&self) -> String {
+impl ToString for Hook {
+    fn to_string(&self) -> String {
```

Comment:
instead of just having a to_string function, implement the ToString trait

https://rust-lang.github.io/rust-clippy/master/index.html#inherent_to_string